marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/4/impl/flow-lib.xqy

```diff
@@ -1023,7 +1026,7 @@ declare function flow:safe-run($func)
     catch($ex) {
       debug:log(xdmp:describe($ex, (), ())),
       trace:error-trace($rfc:item-context, $ex, xdmp:elapsed-time() - $before),
-      fn:error((), "DATAHUB-PLUGIN-ERROR", $ex)
+      httputils:throw-bad-request((), $ex)
```

Comment:
@rjrudin ,
This change and a similar change in flow-lib.sjs in dhfprod-6284 are causing a number of test failures (These failures are for changes in flow-lib.sjs https://jenkins.marklogic.com/blue/organizations/jenkins/Datahub_CI/detail/develop/1692/tests/  alone). Should we not make this change in dhf 4 modules ? I am seeing that no matter a 4.x flow flow succeeds or fails, it always returns a http 200.

Reply:
@srinathgit If a 200 is always returned, then yeah - no need to change these. Can you add a comment at the top of flow-lib.xqy and flow-lib.sjs describing why we're not using the httpUtils functions so future developers are aware?